### PR TITLE
Fixes for Video support

### PR DESCRIPTION
**Pull request by igorcferreira**
This PR tackles 3 major issues in the module that is parsing Jingle->SDP: 

- **[Fixes the parse of the muxing information for data channel](https://github.com/FutureWorkshops/XMPPFramework/compare/release/stc...FutureWorkshops:feature/video_call#diff-9ef69cf56a1472f062398288c7edf8aaR110)**
- **[Add the parse for source groups on media](https://github.com/FutureWorkshops/XMPPFramework/compare/release/stc...FutureWorkshops:feature/video_call#diff-9ef69cf56a1472f062398288c7edf8aaR181)**
- **[Improve source-add parse & operation](https://github.com/FutureWorkshops/XMPPFramework/compare/release/stc...FutureWorkshops:feature/video_call#diff-479d692434428b9139d91be117ff03cdR366)**

### CODE REVIEW

**Comment by igorcferreira** [comment]
@fwroberto I created by accident as open (not draft), but please do not merge. Since this changes the XMPP Framework version.

File: Extensions/XEP-0166/XMPPJingleSDP.m [hunk 1]
```diff
@@ -540,6 +558,9 @@ - (XMPPIQ *)SDPToXMPP:(NSString *)sdp action:(NSString *)action initiator:(XMPPJ
         if (media_name == nil || [media_name isEqualToString:@""] || [media_name isEqualToString:@"application"]) {
             media_name = @"data";
         }
+        if ([media_name isEqualToString:@"data"]) {
+            continue;
+        }
```
Comment (fwroberto) [inline Extensions/XEP-0166/XMPPJingleSDP.m]: @igorcferreira is this @"data" some constant we should put in a variable or parameterise somewhere? I see it being used in multiple places throughout the file

File: Extensions/XEP-0166/XMPPJingleSDP.m [hunk 2]
```diff
@@ -895,68 +936,72 @@ - (XMPPIQ *)CandidateToXMPP:(NSDictionary *)dict media:(NSString *)media action:
...
 
 }
+
+- (NSXMLElement *)parseCandidate:(NSDictionary *)candidate forMedia:(NSString *)media {
+    
+    NSXMLElement *contentElement = [NSXMLElement elementWithName:@"content"];
+    [contentElement addAttributeWithName:@"creator" stringValue:@"responder"];
+    [contentElement addAttributeWithName:@"name" stringValue:media];
+    
+    //NSXMLElement *transElement = transElement1;
```
Comment (fwroberto) [inline Extensions/XEP-0166/XMPPJingleSDP.m]: @igorcferreira can we remove this or is it useful?

Comment (igorcferreira) [reply Extensions/XEP-0166/XMPPJingleSDP.m]: @fwroberto Done

File: Extensions/XEP-0166/XMPPJingleSDP.m (hunk 1 above)
Comment (igorcferreira) [reply Extensions/XEP-0166/XMPPJingleSDP.m]: @fwroberto Done

**Review by fwroberto — APPROVED** [review]
LGTM

**Pull request merged**